config/routes.rb

```diff
@@ -9,9 +9,18 @@
     get "/login", to: "sessions#new"
     post "/login", to: "sessions#create"
     delete "/logout", to: "sessions#destroy"
-    resources :users
+    get "activation/:token", to: "sessions#edit", as: "activation"
+    get ":id/following", to: "following#index", as: "following"
```

Comment:
it's  still work ok when remove this line right?

Reply:
I put it below but not remove it...

Reply:
let's remove it